### ISSUE TITLE
[0.9.0 Staging-1501] Skid's moved inventory keeps moving back

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoIssues#15662]
Ran into a strange bug in the new build (wasn't an issue in the version before 1501). Because of #15525 I sometimes empty my skid by going into its inventory and dragging items out of it into the Truck. This works pretty well. However, today when I did that, the items moved, but when I started digging, the items moved back into the skid and I was out of inventory again. It wouldn't revert if you moved the items to a stockpile from the Truck.

1) Version
0.9.0 Staging-1501

2) Steps to reproduce
- (be on the test server?)
- Dig with a Skid, get it full
- Open Skid's inventory
- Drag items from Skid's inventory into a Truck
- Resume digging with a Skid
- Skid is now instantly full, Truck is empty
- Drag items from Skid again, deposit items from Truck to Stockpile
- Resume digging with a Skid, it works fine

Video: https://youtu.be/3-svJV92hKw

3) Expected behavior
Items should stay in Truck.

4) Actual behavior
Items move back from the Truck to Skid

5) Providing Log files
https://mega.nz/file/8EQQFITL#O1JgcSDXX2RX-7NIM3F6u1mCSy8vr1u2dkYX--0NCh4

**Comment by beastyn** [comment]
Can't reproduce yet
Storage error from the log
```
Coroutine couldn't be started because the the game object 'Storage' is inactive!
UI.StorageUI:UpdateLinked()
UnityEngine.AndroidJavaRunnable:Invoke()
UI.StorageUI:InitLinked(LinkComponentView)
UI.StorageUI:OnInitPanel()
UI.WorldObjectUI:InitPanel(String)
UI.WorldObjectUI:SetPanelEnabled(PanelData, Boolean)
System.Action`1:Invoke(T)
UnityEngine.Events.InvokableCall`1:Invoke(T1)
UnityEngine.Events.UnityEvent`1:Invoke(T0)
UnityEngine.UI.Toggle:Set(Boolean, Boolean)
UI.WorldObjectUI:AutoSelectPanel()
UI.WorldObjectUI:UpdateSelectedPanel()
UnityEngine.AndroidJavaRunnable:Invoke()
UI.WorldObjectUI:Init(BSONObject)
UI.UIManager:Open(String, BSONObject, UILayer)
GenericClient`1:OpenUI(String, BSONObject)
System.Reflection.MonoMethod:Invoke(Object, BindingFlags, Binder, Object[], CultureInfo)
System.Reflection.MethodBase:Invoke(Object, Object[])
System.Func`3:Invoke(T1, T2)
Eco.Shared.Networking.RPCManager:TryInvoke(INetClient, Object, String, BSONArray, Object&)
Eco.Shared.Networking.RPCManager:InvokeOn(INetClient, BSONObject, Object, String)
Eco.Shared.Networking.RPCManager:HandleReceiveRPC(INetClient, BSONObject)
NetworkManager:Eco.Shared.Networking.INetworkEventHandler.ReceiveEvent(INetClient, NetworkEvent, BSONObject)
Eco.Shared.Networking.NetworkClient:TryReceivePacket()
Eco.Shared.Networking.ClientPacketHandler:ReceiveWhile(NetworkClient, Func`1)
System.Action`1:Invoke(T)
FramePlanner.PlannerGroup:OnUpdate()
FramePlanner.FramePlannerSystem:OnUpdate()
Unity.Entities.ComponentSystem:InternalUpdate()
Unity.Entities.ComponentSystemGroup:OnUpdate()
Unity.Entities.ComponentSystem:InternalUpdate()
Steamworks.RulesRefreshComplete:Invoke()

```

**Comment by NN-Mike** [comment]
Also seen on #15559

**Comment by SlayksWood** [comment]
merged to #15559
